tests/vc-verifier/index.js

```diff
@@ -0,0 +1,91 @@
+/*!
+ * Copyright 2023 Digital Bazaar, Inc.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
```

Comment:
@BigBlueHat I copied the current file header but LMK if this repo license needs to be changed